<html lang="en-US">
 <body>
  <h2 id="	">            
    <ul>
    <font size="8">Protocol:<a href="https://github.com/yaqike/loquat/blob/main/lab-pipeline.sh">Pipeline</font></a>
    <h5 align="left"> </h5>
      <ul>
      <li>QIIME 2&nbsp&nbsp&nbsp<a href="https://github.com/yaqike/loquat/tree/main/qiime2">Example Datasets</a></li>
      <font size="3">[1]Bolyen E, Rideout JR, Dillon MR, et al. Reproducible, interactive, scalable and extensible microbiome data science using QIIME 2. Nat Biotechnol. 2019;37(8):852-857. doi:10.1038/s41587-019-0209-9</font>
      <h3 align="left"> </h3>

      <li>PICRUSt2&nbsp&nbsp&nbsp<a href="https://github.com/yaqike/loquat/tree/main/picrust2">Example Datasets</a></li>
      <font size="3">[2]Douglas GM, Maffei VJ, Zaneveld JR, et al. PICRUSt2 for prediction of metagenome functions. Nat Biotechnol. 2020;38(6):685-688. doi:10.1038/s41587-020-0548-6</font>
      <h3 align="left"> </h3>

      <li>MicrobiomeAnalyst&nbsp&nbsp&nbsp<a href="https://github.com/yaqike/loquat/tree/main/MicrobiomeAnalyst">Example Datasets</a></li>
      <font size="3">[3]Chong J, Liu P, Zhou G, Xia J. Using MicrobiomeAnalyst for comprehensive statistical, functional, and meta-analysis of microbiome data. Nat Protoc. 2020;15(3):799-821. doi:10.1038/s41596-019-0264-1</font>
      <h3 align="left"> </h3>
       
      <li>ggClusterNet&nbsp&nbsp&nbsp<a href="https://github.com/yaqike/loquat/tree/main/ggClusterNet">Example Datasets</a></li>
      <font size="3">[4]Wen T, Xie P, Yang S, et al. ggClusterNet: An R package for microbiome network analysis and modularity-based multiple network layouts. Imeta. 2022;1(3):e32. Published 2022 Jun 13. doi:10.1002/imt2.32</font>
      <h3 align="left"> </h3>
      </ul>
    </ul>
    <br>
    <br>
    <h5 align="center">Powered by <a href="https://orcid.org/0009-0007-9600-5584">jialh</h5> 
 </body>
</html>

